src/DwhManager.php

```diff
@@ -405,16 +400,12 @@ private function ensureUserExists(string $userName, string $type, array $options
                 $type,
                 $options,
             );
-        } else {
-            $this->connection->alterUser(
-                $userName,
-                $options,
-            );
-            $this->logger->info(sprintf(
-                'User "%s" already exists',
-                $userName,
-            ));
+
+            return;
```

Comment:
Jako skoro bych řekl, že by bylo logičtější udělat ten early return opačně - pro stav, kdy existuje a jsou to dva řádky. Ale asi je to jedno